index.js

```diff
@@ -135,6 +135,14 @@ function compression(options) {
         return
       }
 
+      // Don't compress for Cache-Control: no-transform
+      // https://tools.ietf.org/html/rfc7234#section-5.2.1.6
+      var cacheControl = res.getHeader('Cache-Control') || ''
```

Comment:
I did have a look around for an existing lib, but didn't spot anything relevant.